distributed/stealing.py

```diff
@@ -476,10 +474,9 @@ def maybe_move_task(
                             break
                         thief = thieves[i % len(thieves)]
 
-                        duration = victim.processing.get(ts)
-                        if duration is None:
-                            stealable.discard(ts)
-                            continue
+                        duration = self.scheduler.get_task_duration(
+                            ts
+                        ) + self.scheduler.get_comm_cost(ts, ts.processing_on)
```

Comment:
All the stealing fixes here are preliminary. I suspect we want to get https://github.com/dask/distributed/pull/7026 done first